01-messaging.md

```diff
@@ -284,7 +283,9 @@ The sending node:
   - SHOULD set `remote_addr` to reflect the remote IP address (and port) of an
     incoming connection, if the node is the receiver and the connection was done
     via IP.
-  - SHOULD NOT set private addresses as `remote_addr`.
+  - if it sets `remote_addr`:
+     - MUST set it to a valid `address descriptor` (1 byte type and data) as described in [BOLT 7](07-routing-gossip.md#the-node_announcement-message).
+    - SHOULD NOT set private addresses as `remote_addr`.
```

Comment:
nit: this isn't aligned with the other above

Reply:
Yep, the latter was correct (2 spaces per indent), so I fixed the upper.